### PR TITLE
[WIP] Addresses #43 & #44

### LINKED ISSUE #43: Bug Report: RPC Not Connected Error, Multiple RPC Connection Pointers?
## Brief summary of issue

It seems if the package is built from commit 60e4e25a839bd521c5ccee4e99c8f83914f8a783 everything is good; however if the package is built from the current `HEAD` of `master` (7b9cf050c63dae5f3a9b5803389130cff2f039fa) when trying to use any python tool from a downstream repo that opens an rpc connection an rpc not connected error is thrown.

### Types of issue

- [X] Bug report (report an issue with the code)
- [ ] Feature request (request for change which adds functionality)

## Expected Behavior

An rpc connection error should not be thrown.

## Current Behavior

On commit 60e4e25a839bd521c5ccee4e99c8f83914f8a783:

```bash
% confChamber.py -c eagle60 -g3 --vfatmask=0x7f3f3f --run                        
2019.01.15.14.46
Open pickled address table if available  /opt/cmsgemos/etc/maps/amc_address_table_top.pickle...
Initializing AMC eagle60
opened connection
Configuring VFATs with chamber_vfatDACSettings dictionary values
biased VFATs
Set CFG_THR_ARM_DAC to 100
VFATs set to run mode
Chamber Configured
```

On commit 7b9cf050c63dae5f3a9b5803389130cff2f039fa we get:

```
% confChamber.py -c eagle60 -g3 --vfatmask=0x7f3f3f --run                        
Open pickled address table if available  /opt/cmsgemos/etc/maps/amc_address_table_top.pickle...
Initializing AMC eagle60
opened connection
Caught NotConnectedException: Not connected to rpc service
```

### Steps to Reproduce (for bugs)

1. Build `reg_utils/python/reg_interface` package from current `HEAD`,
2. Call any downstream python tool

## Possible Solution (for bugs)

I recall this being due to some `import *` shenanigans last time.

There's probably an import that has changed somewhere that causes an additional rpc pointer to be loaded by mistake.

## Context (for feature requests)

Breaks python API.

Unable to use the new `set-phase` feature of `gbt.py`.

### PR DESCRIPTION
**Pull request by bdorney**
## Description

Addresses #43 by reverting 4a9ceb68ed78645c4dd1a4ebdc1d88ab04d53a86.
Addresses #44 by removing duplicate function.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context

Resolves #43 & #44

## How Has This Been Tested?

WIP

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: python/reg_interface/common/reg_base_ops.py [hunk 1]
```diff
@@ -12,7 +12,10 @@
   wReg = lib.putReg
   wReg.argtypes=[c_uint,c_uint]
 else:
-  lib = CDLL("librwreg.so")
+  try:
```
Comment (mexanick) [inline python/reg_interface/common/reg_base_ops.py:18]: Please insert here a comment that this has to be removed in future after proper refactoring.
As I can recall, the issue is that our gem-specific tools has been moved out of reg_utils to keep it basic, but it happened after the legacy branch off, that's why the discrepancy appeared.

Comment (bdorney) [reply python/reg_interface/common/reg_base_ops.py:18]: Done

**Comment by jsturdy** [comment]
If this is still WIP, can you `cherry-pick` from #46 onto the correct base branch?
Otherwise, let's just mark this as superseded, close it, and handle everything in #46

**Comment by bdorney** [comment]
Superseded by #46

**Pull request closed without merging**